README.md

```diff
@@ -119,37 +119,76 @@ returns
 }
 ```
 
-### Create Event
-POST /api/events/
-```
-{
-  "name": "<event_name>"
-}
-```
+### Get all tasks for a user
+GET /api/tasks/me
+
 returns
 ```
 {
   "data": {
-    "id": <event_id>,
-    "created_at": <timestamp>
-    "name": "<event_name>"
+    "not_scheduled": [
+      {
+        "id": <task_id>,
+        "name": "<task_name>",
+        "description": "<task_description>",
+        "duration": <task_duration_in_seconds>
+        "scheduled": <boolean>,
+        "completed": <boolean>,
+        "user_id": <TWAIN_USER_ID>
+      },
+      ...
+    ], 
+    "scheduled": [
+      {
+        "id": <task_id>,
+        "name": "<task_name>",
+        "description": "<task_description>",
+        "duration": <task_duration_in_seconds>
+        "scheduled": <boolean>,
+        "completed": <boolean>,
+        "user_id": <TWAIN_USER_ID>
+      },
+      ...
+    ]
   },
   "error": "<ERROR_MESSAGE>"
 }
 ```
+**Note:** 
+  *`not_scheduled` is ordered by most recently added at the top
+  *`scheduled` is ordered by increasing chronological order
 
-### Get event by ID
-GET /api/events/{id}
-
+### Set Tasks Complete by ID (Array of IDs) 
+POST /api/tasks/complete_task
+```
+{
+	"ids": [<array_of_task_ids>]
+}
+```
 returns
 ```
 {
-  "data": {
-    "id": <event_id>,
-    "created_at": <timestamp>
-    "name": "<event_name>"
-  },
-  "error": "<ERROR_MESSAGE>"
+    "data": [
+        {
+            "id": <event_id>,
+            "name": "<event_name>",
+            "description": "<event_description>",
+            "duration": <task_duration_in_seconds>,
+            "scheduled": <boolean>,
+            "completed": <boolean>,
+            "timecompleted": "<timestamp>"
+        },
+        {
+            "id2": <event_id2>,
+            "name2": "<event_name2>",
+            "description2": "<event_description2>",
+            "duration2": <task_duration_in_seconds2>,
+            "scheduled2": <boolean2>,
+            "completed2": <boolean2>,
+            "timecompleted2": "<timestamp2>"
```

Comment:
I think this can be confusing. Someone reading our docs in the future might interpret _id2_ as a different field from _id_, even though I know they're the same and calling it _id2_ is just referring to the fact that it's the second item to be returned. Personally, I would replace the second event object with ellipses to show that the elements of _data_ keep going.